### PR TITLE
Fix guided tour popup margins

### PR DESCRIPTION
**Pull request by nielslange**
## Description

Removed negative left and right margins from guided tour popup when scrolling on mobile devices.

## How has this been tested?

Fix had been tested using iPhone 7 Plus with iOS 11.4.1 in both English (LTR) and Hebrew (RTL).

## Types of changes

Removed two negative margins (LTR & RTL) within `gutenberg/packages/components/src/popover/style.scss`

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation.

### CODE REVIEW

File: packages/components/src/popover/style.scss [hunk 1]
```diff
@@ -171,7 +171,12 @@ $arrow-size: 8px;
 
 	.components-popover:not(.is-mobile):not(.is-middle).is-right & {
 		/*!rtl:ignore*/
-		margin-left: -24px;
+		margin-left: 0;
+	}
+
+	.components-popover.edit-post-more-menu__content:not(.is-mobile):not(.is-middle).is-right & {
+		/*!rtl:ignore*/
```
Comment (tofumatt) [inline packages/components/src/popover/style.scss]: Could you add some comments explaining why these should be ignored? Maybe even just a link to this PR, just so future developers will understand.

**Review by tofumatt — CHANGES_REQUESTED** [review]
Thanks for the patch!

I think this makes sense and looks good, but I'm a bit confused about the screenshots you've submitted. It looks like there are "before" and "after" screenshots, but I can't tell which ones are which so it's hard to tell what the change is here.

Could you edit your PR comment to clarify the before/after screenshots and have a look at the comments I made? After that this should be good to merge 😄

**Comment by nielslange** [comment]
@tofumatt Thanks for your feedback. I made the screenshots smaller and added title tags to it, thus, when you hover over the images, you'll see their titles and will reply to your other notes directly in the related sections.

File: package-lock.json [hunk 2]
```diff
@@ -2415,6 +2415,7 @@
 			"version": "file:packages/rich-text",
 			"requires": {
 				"@babel/runtime": "^7.0.0",
+				"@wordpress/escape-html": "file:packages/escape-html",
```
[the remark this answers is not in the document]
Comment (nielslange) [reply package-lock.json]: Not sure how this ended up here, but I'll remove it.

File: packages/components/src/popover/style.scss (hunk 1 above)
Comment (nielslange) [reply packages/components/src/popover/style.scss]: `*!rtl:ignore*/` came from the CSS statement before, which I copied and adjusted.

File: package-lock.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (nielslange) [reply package-lock.json]: Just removed it and added commit [25ce95d](https://github.com/WordPress/gutenberg/pull/10295/commits/25ce95d0823bbeefb7e3608fff190b27180c4816).

[the remark this answers is not in the document]
Comment (tofumatt) [reply package-lock.json]: It wasn't your fault! (See: #10304) 😄

**Comment by tofumatt** [comment]
Nope, sorry, just didn't see this was ready for another review; looking now!

**Review by tofumatt — APPROVED** [review]
Those are some epic selectors, but it was like that already, so 🤷‍♂️

Thanks for the patch! Once Travis passes I'll get things merged.

**Comment by nielslange** [comment]
Great to see this PR got merged, @tofumatt! It's my first contribution to Gutenberg and most likely not the last. 😀

**Pull request merged**